### PR TITLE
add back changes to test UI

### PR DESCRIPTION
**Pull request by tballmsft**

### CODE REVIEW

File: src/components/ServiceTest.tsx [hunk 1]
```diff
@@ -76,14 +80,13 @@ function TestStatusIcon(props: { test: JDTestRunner }) {
 
 function TestListItem(props: { test: JDTestRunner }) {
     const { test } = props;
-    const { specification } = test;
-    const { description } = specification;
+    const { output } = test;
```
Comment (pelikhan) [inline src/components/ServiceTest.tsx:83]: how do you track changes in the output? use ``useChange(test, t => t.outout)``

File: src/components/ServiceTest.tsx [hunk 2]
```diff
@@ -100,45 +103,74 @@ function TestList(props: { testRunner: JDServiceTestRunner }) {
...
+            return <PlayCircleFilledIcon color="action" />
+        case JDCommandStatus.Failed: return <ErrorIcon color="error" />
+        case JDCommandStatus.Passed: return <CheckCircleIcon color="primary" />
+        default: return <HourglassEmptyIcon color="disabled" />
+    }
+}
+
+function CommandListItem(props: { command: JDCommandRunner }) {
+    const { command } = props;
+    const { prompt, progress } = useChange(command, c => c.output);
```
Comment (pelikhan) [inline src/components/ServiceTest.tsx:122]: can output be undefined? you need to guard about this. why is prompt part of output?

File: src/components/ServiceTest.tsx [hunk 3]
```diff
@@ -100,45 +103,74 @@ function TestList(props: { testRunner: JDServiceTestRunner }) {
...
+    const status = useChange(command, c => c.status);
+    const handleAnswer = (status: JDCommandStatus) => () => command.finish(status)
+    return <ListItem>
+        <ListItemIcon>
+            <CommandStatusIcon command={command} />
+        </ListItemIcon>
+        <ListItemText primary={prompt} secondary={!progress ? "" : progress.toString()} />
```
Comment (pelikhan) [inline src/components/ServiceTest.tsx:129]: maybe ``secondary={progress?.toString()}``. what is progress?

File: src/components/ServiceTest.tsx [hunk 4]
```diff
@@ -76,14 +80,13 @@ function TestStatusIcon(props: { test: JDTestRunner }) {
 
 function TestListItem(props: { test: JDTestRunner }) {
```
Comment (pelikhan) [inline src/components/ServiceTest.tsx:81]: not used anywhere?

**Comment by pelikhan** [comment]
We can tidy up later once the plumbing works.

File: src/components/ServiceTest.tsx (hunk 3 above)
Comment (tballmsft) [reply src/components/ServiceTest.tsx:129]: depending on the test type, progress records (from 0.0 to 1.0) how close the test is to succeeding (1.0 means passed).

File: src/components/ServiceTest.tsx (hunk 4 above)
Comment (tballmsft) [reply src/components/ServiceTest.tsx:81]: your code - it is used.

File: src/components/ServiceTest.tsx [hunk 5]
```diff
@@ -100,45 +103,74 @@ function TestList(props: { testRunner: JDServiceTestRunner }) {
...
     const { test } = props;
-    const { specification } = test;
-    const { description } = specification;
+    const { output } = test;
 
     return <Card>
         <CardContent>
-            <Typography variant="h5" component="h2">{description}</Typography>
-            <TestStepper
+            <Typography variant="h5" component="h2">{output}</Typography>
```
[the remark this answers is not in the document]
Comment (tballmsft) [reply src/components/ServiceTest.tsx]: renamed "output" to "description. h5 is your code

**Pull request merged**